Makefile

```diff
@@ -126,13 +126,29 @@ build:
 		.
 
 ################################################################################
-# Scan                                                                         #
+# Image security                                                               #
 ################################################################################
 
 .PHONY: scan
 scan:
 	grype $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG) -f medium
 
+.PHONY: generate-sbom
+generate-sbom:
+	syft $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG) \
+		-o spdx-json \
+		--file ./artifacts/brigade-acr-gateway-$(VERSION)-SBOM.json
+
+.PHONY: publish-sbom
+publish-sbom: generate-sbom
+	ghr \
```

Comment:
So is this actually creating the initial release for ${VERSION} and attaching the artifact?  Or is it amending the release?  (Or other?)

Reply:
I believe it would create an initial release if we hadn't already created one, but if it's an existing release, it amends it with artifacts. It's the same as how we publish the `brig` CLI in the main repo.